frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/SelectedNodeVolumeMountsTab.tsx

```diff
@@ -28,7 +28,6 @@ const SelectedNodeVolumeMountsTab: React.FC<SelectedNodeVolumeMountsTabProps> =
     <>
       {renderDetailItems(
         Object.values(items).map(({ name, mountPath }) => ({ key: mountPath, value: name })),
-        true,
```

Comment:
Do you have a screenshot of how this change effected the vollume mount tab?

Reply:
No, and I'm not sure how I would go about demoing this tab.